tests/typowalking/wordwalker.js

```diff
@@ -27,7 +28,8 @@ bender.test( {
 		wordwalker = new editor.plugins.nanospell.WordWalker(range);
 
 		while (currWordObj = wordwalker.getNextWord()) {
-			wordsReturned.push(currWordObj.word);
+			word = currWordObj.word;
+			if (word) wordsReturned.push(word);
```

Comment:
I wouldn't spend too much time on this test though.  If it's not obvious to set up...